igrafx_extension/igrafx_knime_extension/igx_env_osx_arm.yml

```diff
@@ -82,15 +82,4 @@ dependencies:
   - zipp=3.17.0=pyhd8ed1ab_0
   - zstd=1.5.5=h4f39d0f_0
   - pip:
-      - certifi==2023.11.17
-      - charset-normalizer==3.3.2
-      - idna==3.6
-      - igrafx-mining-sdk==2.28.0
-      - networkx==2.6.3
-      - pydruid==0.6.5
-      - python-dotenv==0.19.2
-      - requests==2.29.0
-      - sqlalchemy==2.0.12
-      - toml==0.10.2
-      - typing-extensions==4.8.0
-      - urllib3==1.26.18
+      - igrafx-mining-sdk==2.28.1
```

Comment:
Have you tested if it works? I am not sure that extension can be build for Mac without explicit definition in this yml of all dependencies.